app/assets.py

```diff
@@ -30,3 +30,23 @@
     filters='jsmin',
     output='scripts/vendor.js'
 )
+
+asylum_scss = Bundle(
+    'asylum/main.css',
```

Comment:
Very good catch. This was on my list of things to investigate but I forgot to. There are `.scss` files for the asylum connect catalog styling, but I couldn't get Flask to see them and compile them properly. `main.css` is actually the result of compiling the scss. Ideally this bundle would refer to the `scss` files, apply the filter and just style with that, but I couldn't get that to work for whatever reason.